localization_ui/src/main/res/layout/fragment_language_settings.xml

```diff
@@ -114,7 +114,7 @@
                     android:layout_marginEnd="@dimen/marginDouble"
                     android:background="?attr/languageSettingsButtonBackground"
                     android:button="@null"
-                    android:text="@string/english"
+                    android:text="@string/english_in_that_language"
```

Comment:
This isn't the dropdown view, should we be changing this view?

Reply:
This requires a different resource. Since I redefined `@string/english`, `@string/spanish`, and `@string/chinese`, this was the term I used for the untranslatable text. I can rename to `@string/english_in_english` etc. if that helps?